pkg/domain/entities/container_ps.go

```diff
@@ -46,13 +50,69 @@ type ListContainer struct {
 	// boolean to be set
 	PodName string
 	// Port mappings
-	Ports []ocicni.PortMapping
+	PortMappings []ocicni.PortMapping
 	// Size of the container rootfs.  Requires the size boolean to be true
-	Size *shared.ContainerSize
+	ContainerSize *shared.ContainerSize
 	// Time when container started
 	StartedAt int64
 	// State of container
-	State string
+	ContainerState string
+}
+
+// State returns the container state in human duration
+func (l ListContainer) State() string {
+	var state string
+	switch l.ContainerState {
+	case "running":
+		t := units.HumanDuration(time.Since(time.Unix(l.StartedAt, 0)))
+		state = "Up " + t + " ago"
+	case "configured":
+		state = "Created"
+	case "exited":
+		t := units.HumanDuration(time.Since(time.Unix(l.ExitedAt, 0)))
+		state = fmt.Sprintf("Exited (%d) %s ago", l.ExitCode, t)
+	default:
+		state = l.ContainerState
+	}
+	return state
+}
+
+// Command returns the container command in string format
+func (l ListContainer) Command() string {
+	return strings.Join(l.Cmd, " ")
+}
+
+// Size returns the rootfs and virtual sizes in human duration in
+// and output form (string) suitable for ps
+func (l ListContainer) Size() string {
+	virt := units.HumanSizeWithPrecision(float64(l.ContainerSize.RootFsSize), 3)
+	s := units.HumanSizeWithPrecision(float64(l.ContainerSize.RwSize), 3)
+	return fmt.Sprintf("%s (virtual %s)", s, virt)
+}
+
+// Names returns the container name in string format
+func (l ListContainer) Names() string {
```

Comment:
Do we need the plural on this method since it is returning a Name rather than a slice?

Reply:
Names is the go template key  ... thats what v1podman and docker use